src/video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -57,7 +75,19 @@ def analyze_video(video_path, time_range=None):
     return dict_of_faces
 
 
-def analyze_frame(frame):
+def analyze_frame(frame) -> [[int, int]]:
+    """Analyses a single frame finding faces returning as a coordinate
+
+    Parameters
+    ----------
+    frame
+        the frame of the video taken from cap.read()
+
+    Returns
+    -------
+    [[int, int]]
+        A list of facetuples that describe the placement of a face in the frame
+    """
     # Get width and height of frame
```

Comment:
There should be an empty line between the documentation and the first line of code.